.github/workflows/release-tauri.yml

```diff
@@ -0,0 +1,89 @@
+name: Release Tauri
```

Comment:
This works and will create a separate tag and release for tauri, for now. We can, instead, simply eventually rename this action, in a follow-up PR, to "Release", and have it create the release and upload everything it can (electron, cordova, web extensions, etc.), automatically, and then we just need to "finalize it" (publish as not pre-release in GitHub), then download and upload the files to the respective stores.